### ISSUE TITLE
Standardize all loggers to include guild

### ISSUE THREAD

**Issue by aaronrai24** [aaronrai24/DollarDiscordBot#40]
Feature request: Since Dollar's user base is growing its becoming increasingly difficuly to know what discord loggers are coming from, lets use the guild object of ctx to include more info

Submitted by: Cash#8915
Server: mfDiscord

**Comment by aaronrai24** [comment]
This has been changed to be standardize loggers per commands, adding guild isn't possible since the logger gets created once, altering doesn't seem possible, but even if it is that would require us to pass guild each time a command is used or some event happens and that isn't practical. By default the logging level is set at INFO when Dollar starts up. I added !logging (level) that allows us to change logging level during runtime. 
New loggers:
- `music`: Relates to music related commands and functions
- `game-commands`: Relates to game commands
- `watchlist`: Relates to watchlist commands
- `core`: General logger that is used in Dollar's core functions that are used repeatedly
- `diagnostic`: Relates to Dollar diagnostic commands that are ADMIN only. Ensures diagnostic commands are working properly
- `administrative`: Relates to administrative commands
